src/browser/main.js

```diff
@@ -1888,7 +1894,7 @@ import { log_data } from "../log.js";
                 settings.initrd = { buffer: initrd };
             }
 
-            const title = multiboot?.name || hda?.name || cdrom?.name || hdb?.name || fda?.name || bios?.name;
+            const title = multiboot?.name || hda?.name || cdrom?.name || hdb?.name || fda?.name || bios?.name || vga_bios?.name;
```

Comment:
This code selects the first boot medium as the title. Since the vgabios is not bootable on its own, I don't think it should be here.